corehq/apps/app_manager/util.py

```diff
@@ -639,12 +639,12 @@ def get_latest_enabled_build_for_profile(domain, profile_id):
 
 
 @quickcache(['domain', 'location_id', 'app_id'], timeout=24 * 60 * 60)
-def get_latest_enabled_app_release(domain, location_id, app_id):
+def get_app_release_by_location(domain, location_id, app_id):
```

Comment:
This one actually is the latest enabled release, right?

Reply:
[renamed](https://github.com/dimagi/commcare-hq/pull/23680/commits/a1bc83cc1f244c5f4e4acf0e5b20f0b7ac8934e1)